pkg/storj/node.go

```diff
@@ -143,6 +144,19 @@ func (id NodeID) MarshalJSON() ([]byte, error) {
 	return []byte(`"` + id.String() + `"`), nil
 }
 
+// Value set a NodeID to a database field
+func (id NodeID) Value() (driver.Value, error) {
```

Comment:
might be worth adding a quick test or two for `Value()` and `Scan()`. They look good to me though.

Reply:
Added